internal/role/role.go

```diff
@@ -0,0 +1,41 @@
+// Licensed to Yugabyte, Inc. under one or more contributor license
```

Comment:
This file is repeated here also: https://github.com/yugabyte/ybm-cli/pull/137/files, can we merge these PRs?

Reply:
Same @posriniv make sure you don t miss the merge